qhub_ops/template/{{ cookiecutter.repo_directory }}/scripts/00-guided-install.sh

```diff
@@ -13,6 +13,12 @@ if [ ! -f "$(command -v terraform)" ]; then
     exit 1
 fi
 
+# 02 Check version of Terraform
+if ! terraform --version | grep -q "v0.12.24"; then 
+	echo "Error: Please install Terraform v0.12.24"
+	exit 1
+fi
+
```

Comment:
Can you please add this to docs: https://github.com/Quansight/qhub-ops/tree/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D ?